Jenkinsfile

```diff
@@ -176,6 +176,9 @@ pipeline {
                             junit testResults: "**/*/TEST*${SUREFIRE_REPORT_NAME_SUFFIX}.xml", keepLongStdio: true, allowEmptyResults: true
                             junit testResults: "**/*/TEST*${SUREFIRE_REPORT_NAME_SUFFIX}-FLAKY.xml", keepLongStdio: true, allowEmptyResults: true
                         }
+                        failure {
+                            archiveArtifacts artifacts: '**/FlakyTests.txt, **/DuplicateTests.txt', allowEmptyArchive: true
```

Comment:
Oh, thanks for adding the flaky tests as well!

Reply:
Let's see if this works first :D

Reply:
Hm, if it's best practice I'll change it. Personally, I'm always annoyed when I have to unzip a single text file that's < 10KiB

Reply:
Leave it as is then, I can't remember the reasoning anymore - if it's an issue, we'll find out again :smile: It might've simply been that test reports, uncompressed, can be hundreds of megabytes.